ballerina/Ballerina.toml

```
@@ -16,3 +17,21 @@ groupId = "io.ballerina.lib"
 artifactId = "jsondata-native"
 version = "0.1.0"
 path = "../native/build/libs/data.jsondata-native-0.1.0-SNAPSHOT.jar"
+
+[[platform.java11.dependency]]
+path = "./lib/json-path-2.7.0.jar"
+groupId = "com.jayway.jsonpath"
+artifactId = "json-path"
+version = "2.7.0"
+
+[[platform.java11.dependency]]
+path = "./lib/json-smart-2.4.7.jar"
+groupId = "net.minidev.json"
+artifactId = "json-smart"
+version = "2.4.7"
+
+[[platform.java11.dependency]]
+path = "./lib/accessors-smart-2.4.7.jar"
+groupId = "net.minidev.json"
+artifactId = "accessors-smart"
+version = "2.4.7"
```

```suggestion
[[platform.java11.dependency]]
path = "./lib/json-smart-2.4.7.jar"
groupId = "net.minidev.json"
artifactId = "json-smart"
version = "2.4.7"

[[platform.java11.dependency]]
path = "./lib/accessors-smart-2.4.7.jar"
groupId = "net.minidev.json"
artifactId = "accessors-smart"
version = "2.4.7"
```
Aren't we just need the `jayway` dependency?

---

We need to define other 2 on the Ballerina.toml.
I guess those are not pack with the JsonPath module.